test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests.cs

```diff
@@ -287,7 +302,7 @@ public SamplesEndToEndTests(TestFixture fixture)
             Assert.NotNull(log);
         }
 
-        [Fact(Skip = "Needs investigation")]
+        [Fact(Skip = "http://github.com/Azure/azure-functions-host/issues/2812")]
```

Comment:
Tried to reenable, but ran into issues. Actually a lot of the Node tests still fail due to various language worker issues.